cmd/cli/install.go

```diff
@@ -159,11 +171,19 @@ func newInstallCmd(config *helm.Configuration, out io.Writer) *cobra.Command {
 				return errors.Errorf("Could not access Kubernetes cluster, check kubeconfig: %s", err)
 			}
 			inst.clientSet = clientset
+
+			configClient, err := configClientset.NewForConfig(kubeconfig)
```

Comment:
oh. This messed with my mind so much. Can the package import be renamed something more osm-ish? ie. `osm-config-client` or something.

Reply:
If you find it confusing, so will other people. So yes, will change the name